PraceDomowe/PracaDomowa5/Okreglicki Krystian PD5/Okreglicki Krystian PD5.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>Liczba postów dla których warunek zachodzi jest OK, ale liczba pytań w zbiorze to nie są tylko te Posts dla których istnieje AcceptedAnswer, tylko te, których PostTypeId = 1 (chociaż to trzeba by było odczytać z opisu danych). -0.1 za wynik nie w postaci odsetka. </p><p> </p><p><code>1.9 punktu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>